src/main/java/com/iexec/standalone/feign/LoginService.java

```diff
@@ -34,7 +34,7 @@
 public class LoginService {
 
     static final String TOKEN_PREFIX = "Bearer ";
-    private String jwtToken;
+    private String jwtToken = "Bearer ";
```

Comment:
If you wish to keep the space, this could become
```suggestion
    private String jwtToken = TOKEN_PREFIX;
```